tests/api/test_processes.py

```diff
@@ -179,3 +227,35 @@ async def test_get_process_by_process_id(test_client, mirror_workflow_process):
 async def test_get_process_by_process_id_404(test_client):
     response = test_client.get("/processes/mirror-test")
     assert response.status_code == 404
+
+
+@pytest.mark.asyncio
+async def test_post_valid_name_valid_payload(test_client, process_payload_valid):
+    response = test_client.post(
+        "/processes/mirror/execution", data=json.dumps(process_payload_valid)
+    )
+    assert response.status_code == 200
```

Comment:
Per the spec, we should be returning a 201 for an async response (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201).

Reply:
Fixed